x-pack/plugin/slm/src/test/java/org/elasticsearch/xpack/slm/SnapshotLifecycleServiceTests.java

```diff
@@ -454,13 +455,13 @@ public void testStoppedPriority() {
                 )
             )
         );
-        final SetOnce<ClusterStateUpdateTask> task = new SetOnce<>();
+        final SetOnce<OperationModeUpdateTask> task = new SetOnce<>();
         ClusterService fakeService = new ClusterService(Settings.EMPTY, clusterSettings, threadPool, null) {
             @Override
             public void submitUnbatchedStateUpdateTask(String source, ClusterStateUpdateTask updateTask) {
                 logger.info("--> got task: [source: {}]: {}", source, updateTask);
                 if (updateTask instanceof OperationModeUpdateTask) {
-                    task.set(updateTask);
+                    task.set((OperationModeUpdateTask) updateTask);
```

Comment:
Here you could use the pattern matching feature:
```
if (updateTask instanceof OperationModeUpdateTask operationModeUpdateTask) {
    task.set(operationModeUpdateTask);
}
```

Reply:
Ah cool, didn't know that feature yet, thanks!